src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4604,6 +4604,29 @@ class C
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestSuppressNullableWarningExpression(completionImplementation As CompletionImplementation) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(completionImplementation,
+                              <Document>
+class C
+{
+    void M()
+    {
+        var s = "";
+        s$$
```

Comment:
Should we be concerned about and testing other kinds of expressions to the left of the `!.`, like `Invocation()!.$$`?

Reply:
I hope we should not we do not add a value here.

Reply:
So, usually it's good to have a tiny amount of cases.  Just to make sure.  That strikes a good balance of making sure it's nto a fluke, while not requiring ultra redundant tests with all code combos.

Reply:
note: i'm also ok with just this case :)